### PR TITLE
Feature/footer and fonts

### PR DESCRIPTION
**Pull request by rachelsoae**
What I did:
- Added mixins for header and body font
- Replace font variables with mixins throughout
- Refactored Shopping Bag button to match Checkout page

Did this break anything?
- [x] No
- [ ] Yes

Type of change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor(DRY-ing up/ reorganizing code, etc.)
- [ ] Super small fix (Corrected a typo, removed a comment, etc.)
- [ ] Skip all the other stuff and briefly explain the fix.

Checklist:
- [ ] If this code needs to be tested, all tests are passing
- [ ] The code runs locally
- [ ] There are comments where clarification/ organization is needed
- [ ] The code is DRY. If it's not, I tried my best
- [ ] I reviewed my code before pushing

What's next?
More UI finess

### CODE REVIEW

File: src/styles/_Checkout.scss [hunk 1]
```diff
@@ -4,14 +4,14 @@
 .checkout {
   min-height: 100vh;
   padding: .5rem 2.5rem;
-  font: v.$ternary-font;
+  @include m.body-font;
```
Comment (alivaditis) [inline src/styles/_Checkout.scss:7]: thanks for fixing that

Comment (rachelsoae) [reply src/styles/_Checkout.scss:7]: You're welcome! I'm glad you had the font variable set up already - made it very easy to change.

**Review by alivaditis — APPROVED** [review]
Beautiful!  Fonts look really nice!  Footer lookin good

File: public/index.html [hunk 2]
```diff
@@ -17,7 +17,7 @@
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Amaranth&family=Philosopher:wght@700&family=Spinnaker&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;700&display=swap" rel="stylesheet">
```
Comment (taranveersingh93) [inline public/index.html:20]: thanks for importing multiple weights

File: src/styles/_DescriptionText.scss [hunk 3]
```diff
@@ -10,13 +10,13 @@
...
     margin-bottom: 1rem;
-    font-family: v.$secondary-font;
+    @include m.header-font;
   }
   
   p {
     margin-top: 0;
     margin-bottom: 2rem;
-    font-family: v.$ternary-font;
+    @include m.body-font;
```
Comment (taranveersingh93) [inline src/styles/_DescriptionText.scss:19]: Thank you!

**Review by taranveersingh93 — APPROVED** [review]
Thank you for taking care of it

**Pull request merged**